### ISSUE TITLE
`TypeError` raised at startup

### ISSUE THREAD

**Issue by gipert** [panoptes-organization/panoptes#156]
```console
> panoptes
Traceback (most recent call last):
  File "/home/gipert/.local/bin/panoptes", line 5, in <module>
    from panoptes import main
  File "/home/gipert/.local/pipx/venvs/panoptes-ui/lib/python3.11/site-packages/panoptes/__init__.py", line 1, in <module>
    from .panoptes import main
  File "/home/gipert/.local/pipx/venvs/panoptes-ui/lib/python3.11/site-packages/panoptes/panoptes.py", line 3, in <module>
    from panoptes.app import app
  File "/home/gipert/.local/pipx/venvs/panoptes-ui/lib/python3.11/site-packages/panoptes/app.py", line 10, in <module>
    from panoptes.database import init_db, db_session
  File "/home/gipert/.local/pipx/venvs/panoptes-ui/lib/python3.11/site-packages/panoptes/database.py", line 5, in <module>
    engine = create_engine('sqlite:///.panoptes.db?check_same_thread=False', convert_unicode=True)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<string>", line 2, in create_engine
  File "/home/gipert/.local/pipx/venvs/panoptes-ui/lib/python3.11/site-packages/sqlalchemy/util/deprecations.py", line 281, in warned
    return fn(*args, **kwargs)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^
  File "/home/gipert/.local/pipx/venvs/panoptes-ui/lib/python3.11/site-packages/sqlalchemy/engine/create.py", line 680, in create_engine
    raise TypeError(
TypeError: Invalid argument(s) 'convert_unicode' sent to create_engine(), using configuration SQLiteDialect_pysqlite/QueuePool/Engine.  Please check that the keyword arguments are appropriate for this combination of components.
```

**To Reproduce**
1. `pipx install panoptes-ui`
2. `panoptes`

**Desktop (please complete the following information):**
 - OS: Arch Linux
 - Version: package panoptes-ui 0.2.0, installed using Python 3.11.3

**Comment by gipert** [comment]
Installing `sqlalchemy<2.0` seems to resolve the issue.

**Comment by fgypas** [comment]
Fixed in #157 
Thanks @ryneches
